front-end/src/components/blog/AddTutorial.vue

```diff
@@ -198,7 +198,7 @@ const addArticle = async () => {
 		descriptions: description.value,
 		content: content.value,
 		tags: tags.value,
-		type: 'tutos',
+		type: 'Tuto',
```

Comment:
Use an enum next time, rather than literal types.
Literal types are best lowercase too, but i know why is capitalized.